docs/_documentations/project-actions.md

```diff
@@ -6,35 +6,28 @@ keywords: run, open, import, show, restart, edit, build, logs, tools, project ac
-- **Metrics Dashboard**: Opens the application monitor in the default browser. Use this to monitor the activity and health of your application. This action is only available when the application is running or debugging.
-- **Performance Dashboard**: Opens the performance dashboard in the default browser. This action is available only when the application is running or debugging.
-- **Tekton Dashboard**: This option is available to you if you are using remote connections, and only if you installed pipelines. Use this option to open the dashboard.
-- **Open Container Shell**: Opens a shell into your application container. This action is only available when the container is active.
-
-- **Import Project**: Imports your project into the current workspace.
-- **Show Log Files**: If log files are available, this action displays a list of log files. Click a log file to open it, or click **Show All** to open all available log files. The individual log files are toggle actions. Click the log file again to remove the log file, or click **Hide All** to remove all log files from the **Console** view.
-
-- **Restart in Run Mode**: Restarts the application in run mode. This action is only available on projects that support restart in run mode. 
-- **Restart in Debug Mode**: Restarts the application in debug mode and attaches the debugger. This action is only available on projects that support it. For MicroProfile and Spring projects, a debug session is started automatically. For Node.js projects you are guided in the set up of a debug session in a Chromium based browser. For all other project types that support debug you have to set up a debug session manually. 
-- **Attach Debugger**:  If you accidentally detached the Java debugger or restarted your IDE, use this action to set up a new debug session. This menu item is only available for Java applications that support restart in debug mode. 
-- **Launch Debug Session**: If you ended your Node.js debug session accidentally, use this to set up a new debug session. This menu item is only available for Node.js applications that support restart in debug mode.
-
+The following actions are available by right-clicking the project in the **Codewind Explorer** view. Most actions are available only if the project is enabled.
+
+Some actions open the default browser. If the default browser cannot handle the content, change the default browser.
+
+- **Open Application**: Opens the application in the default browser. This action is available only when the application is running or debugging.
+- **Project Overview**: Opens the overview page for the project. Use this action to see information about the project or edit project settings.
+- **Tekton Dashboard**: This action is available if you are using remote connections and only if you installed pipelines. Use this action to open the dashboard.
```

Comment:
This page is about actions that can be executed using a Codewind project. Thus, Tekton Dashboard shouldn’t be here. It is called “Open Tekton Dashboard” in the IDE. This action appears at the Codewind tree level, and not at the project tree level.